games-engines/easyrpg_libretro/easyrpg_libretro-0.5.1.0_20180216.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A game interpreter to play RPG Maker games, using the libretro API"
+DESCRIPTION="EasyRPG Player is a game interpreter to play RPG Maker 2000, 2003 \
+and EasyRPG games. It uses the LCF parser library (liblcf) to read RPG Maker \
+game data."
+HOMEPAGE="https://easyrpg.org/"
+COPYRIGHT="2007-2018 the EasyRPG team, the libretro team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="80340d4e27b9bed1bf3239e0a0bb941d17e750a8"
+SOURCE_URI="https://github.com/libretro/easyrpg-libretro/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="cf286f0d1cbeb13785443a63d888060b11530869f3002ea8dde30bd07aaedae5"
+SOURCE_FILENAME="libretro-easyrpg-${portVersion/_/-}-$srcGitRev.tar.gz"
+SOURCE_DIR="easyrpg-libretro-$srcGitRev"
+ADDITIONAL_FILES="easyrpg_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	easyrpg_libretro$secondaryArchSuffix = $portVersion
+	addon:easyrpg_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	retroarch$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	"
```

Comment:
It builds fine, and I can also select _"RPG Maker 2000/2003 (EasyRPG)"_ with _"Load Core"_ in retroarch, but the _"Start Core"_ menu does not appear with this core.

Trying to Investigate:
```
~> objdump -x /system/add-ons/libretro/easyrpg_libretro.so | grep NEEDED
  NEEDED               libiconv.so.2
  NEEDED               libstdc++.so.6
  NEEDED               libgcc_s.so.1
  NEEDED               libroot.so
```
Trying a small change in the recipe:
```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
 	retroarch$secondaryArchSuffix
+	lib:libgcc_s$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
+	lib:libstdc++$secondaryArchSuffix
 	"
```
Result: not better :(

BTW, on Haiku, libgcc_s and libstdc++ are provided by the gcc_syslibs package.
(`pkgman search lib:libgcc_s` and `pkgman search lib:libstdc++` shows that gcc_syslibs is the package they belong to.)

Hmm, among all our recipes, the only one which has `lib:libgcc` (in its `REQUIRES`) is blender. And there are absolutely no recipes with `lib:libstdc++` (in its requires too).

So I guess we don't need to add them here. (But I might be wrong.)
I'm writing this in case someone thinks we should start mentioning these dependencies, if that's appropriatte, of course.